sdk/python/kubeflow/training/api/training_client_test.py

```diff
@@ -149,14 +165,14 @@ def __init__(self, kind) -> None:
     ),
     (
         "valid flow",
-        {"job": create_job(), "namespace": "test"},
+        {"job": create_job(), "namespace": TEST_NAME},
```

Comment:
Do you need to change it since we don't verify the namespace name in the test ?

Reply:
I agree, we don't really verify this namespace name, but just for clarity I think it better using variable instead of the same string again and again. WDYT?

Reply:
Sounds good.